backend/firebaseFunctions.js

```diff
@@ -225,6 +225,21 @@ export async function getPosts(circleId) {
 	return posts;
 }
 
+export async function getHiddenPosts() {
+	const UID = await getUIDFromStorage();
+	let hiddenPosts = [];
+	let hiddenPostsIds = await readData(`prayer_circle/users/${UID}/private/hidden_posts`);
```

Comment:
```suggestion
	let hiddenPostsIds = await readData(`prayer_circle/users/${UID}/private/hidden_posts`) || {};
```
this will remove the need for your if statement on `line 232`